backend/src/backend.rs

```diff
@@ -52,7 +50,7 @@ fn __get_dict_value(dict: &[u8], key: &[u8]) -> Option<CLValue> {
 fn __revert(reason: &OdraError) -> ! {
     let code = match reason {
         OdraError::ExecutionError(code, _) => *code,
-        _ => 0
+        _ => 0,
```

Comment:
Code 0 doesn't mean success?

Reply:
We talked with Krzysiek and he will introduce better errors. It will be fixed in the next PR after new error `OdraExecutionError` is available in Odra.